### ISSUE TITLE
Keeping Track of Collections

### ISSUE THREAD

**Issue by vinceparedes** [Access4Learning/NADM#117]
Maintaining the Collections.xsd file is getting to be a nightmare. Making sure that new objects are in the file and deprecated files are removed is a pain. 

I suggest that we move all collection objects to be inline with the single-version of the object. So the pattern for SIF objects would be: 
- the type
- the object
- the collection
- the plural form

This way there is no doubt when an object is added or deleted that all the parts are taken care of. 

Also, this change is to make it easier for spec developers and does not affect software developers (that I can think of).

**Comment by 4pins** [comment]
This would have huge negative impact on software developers.  For instance when I published the infrastructure schemas this way (with roots in different files) I had to field so much criticism that there wasn't one file to point at, I had to create a collections file in the next release.  Additionally our documentation generation takes the same approach, we point at the collections file and are off to the races.
